rpc/core/types/responses.go

```diff
@@ -75,8 +76,8 @@ type ResultValidators struct {
 }
 
 type ResultDumpConsensusState struct {
-	RoundState      string   `json:"round_state"`
-	PeerRoundStates []string `json:"peer_round_states"`
+	RoundState      json.RawMessage   `json:"round_state"`
```

Comment:
Oh - does this make it easier to parse the result?!

Reply:
yeah.. before we're spitting a string (2 strings), now it's a JSON https://github.com/tendermint/tendermint/issues/708#issuecomment-335115500

Reply:
So, it seems like we could use the actual structs here, but we don't want to have to import consensus to get them. So probably we should move some types from `consensus` into `consensus/types` so we can import here. 

Will these raw messages be identical to:

```
type ResultDumpConsensusState struct {
  RoundState cstypes.RoundState
  PeerRoundStates map[string]cstypes.PeerRoundState
}
```

Reply:
Good ideas! Will do now

Reply:
> clients are importing too much crap

that was the reason behind `json.RawMsg`

Reply:
moving types will be a breaking change, though

Reply:
see https://github.com/tendermint/tendermint/pull/724/commits/dc0e8de9b0671fc93d7479b20c2c9b5ea54c48c6. I did not move all of the types. just ones which are necessary